packages/transport/src/modules/messages/MessageController.ts

```diff
@@ -419,7 +419,7 @@ export class MessageController extends TransportController {
         return message;
     }
 
-    private async validateMessageRecipients(recipients: CoreAddress[]) {
+    public async validateMessageRecipients(recipients: CoreAddress[]): Promise<Result<void, CoreError>> {
```

Comment:
```suggestion
    public async validateMessageRecipients(recipients: CoreAddress[]): Promise<Result<void>> {
```
I think the Result already implies that it might be an error.